sqlmesh/cli/main.py

```diff
@@ -109,13 +109,8 @@ def render(
     dialect: t.Optional[str] = None,
 ) -> None:
     """Renders a model's query, optionally expanding referenced models."""
-    snapshot = ctx.obj.snapshots.get(model)
-
-    if not snapshot:
-        raise click.ClickException(f"Model `{model}` not found.")
-
     rendered = ctx.obj.render(
-        snapshot,
+        model,
```

Comment:
This check was moved in `Context.render` (fyi: there wasn't a similar check in the `evaluate` function).